### DISCUSSION TITLE
Is it possible to show direction of a Polyline?

### DISCUSSION THREAD

**Discussion by stevenv-synap** [fleaflet/flutter_map#1766] [Q&A] [answered]
Hi, 
Has anyone achieved directional Polylines? My current solution uses a second layer of Markers above the Polyline which are rotated to the current direction of the endpoint, along the road.
Or, is there a better alternative that ties nicely with the rendering of the line?

Going for something like this:
![image](https://github.com/fleaflet/flutter_map/assets/102791180/ddef0105-43c4-442d-8c9c-72ab9b21822b)

Thanks!

**Comment by josxha** [comment, marked as answer]
Hi @stevenv-synap, using another marker layer on top of the polyline layer is the correct way to go here. 
There is currently no implementation in `flutter_map` that could be used to accieve this.

**Reply by stevenv-synap** [reply]
Thanks @josxha 🙏
